### ISSUE TITLE
How do I actually use nixpak

### ISSUE THREAD

**Issue by DrBanana419** [nixpak/nixpak#111]
Sorry, I'm new to nixos. I set up a minimal flake for my system, and after looking at the example in the README, I've made my flake to look like this: 
`  description = "NixOS System Configuration";
  inputs = {
    # If you want to use the latest upstream version, I recommend using
    # branch "nixos-unstable" instead of "23.11" (the latest stable release).
    nixpkgs.url = "github:Nixos/nixpkgs/nixos-24.05";
   nixpak = {
      url = "github:nixpak/nixpak";
      inputs.nixpkgs.follows = "nixpkgs";
    };
  };
  outputs = { self, nixpkgs, nixpak }: {
    nixosConfigurations = {
      # This should correspond to the hostname of the machine
      hostname = nixpkgs.lib.nixosSystem {
        system = "x86_64-linux";
        modules = [
          ./configuration.nix
          ./hardware-configuration.nix
        ];
      };
    };
packages.x86_64-linux = let
      pkgs = nixpkgs.legacyPackages.x86_64-linux;

      mkNixPak = nixpak.lib.nixpak {
        inherit (pkgs) lib;
        inherit pkgs;
      };

      sandboxed-wget = mkNixPak {
        config = { sloth, ... }: {

          # the application to isolate
          app.package = pkgs.wget;

          # path to the executable to be wrapped
          # this is usually autodetected but
          # can be set explicitly nonetheless
          app.binPath = "bin/wget";

          # enabled by default, flip to disable
          # and to remove dependency on xdg-dbus-proxy
          dbus.enable = true;

          # same usage as --see, --talk, --own
          dbus.policies = {
            "org.freedesktop.DBus" = "talk";
            "ca.desrt.dconf" = "talk";
          };

          # needs to be set for Flatpak emulation
          # defaults to com.nixpak.${name}
          # where ${name} is generated from the drv name like:
          # hello -> Hello
          # my-app -> MyApp
          flatpak.appId = "org.myself.WgetApp";

          bubblewrap = {

            # disable all network access
            network = false;

            # lists of paths to be mounted inside the sandbox
            # supports runtime resolution of environment variables
            # see "Sloth values" below
            bind.rw = [
              (sloth.concat' sloth.homeDir "/Documents")
              (sloth.env "XDG_RUNTIME_DIR")
              # a nested list represents a src -> dest mapping
              # where src != dest
              [
                (sloth.concat' sloth.homeDir "/.local/state/nixpak/wget/config")
                (sloth.concat' sloth.homeDir "/.config")
              ]
            ];
            bind.ro = [
              (sloth.concat' sloth.homeDir "/Downloads")
            ];
            bind.dev = [
              "/dev/dri"
            ];
          };
        };
      };
    in {
      # Just the wrapped /bin/${mainProgram} binary
      wget = sandboxed-wget.config.script;

      # A symlinkJoin that resembles the original package,
      # except the main binary is swapped for the
      # wrapper script, as are textual references
      # to the binary, like in D-Bus service files.
      # Useful for GUI apps.
      wget-env = sandboxed-wget.config.env;
    };
  };
}
`

Basically I pasted the example in the README and replaced hello for wget. Yet, it doesn't seem to get sandboxed.

**Comment by max-privatevoid** [comment]
Aside from the obvious syntactical mistake introduced by yourr copy-pasting (missing `{` at the start), this works fine.

**Comment by DrBanana419** [comment]
When I call `wget <url>` from a shell, it works, despite network being disabled in the sandbox (per the config above). Is there some other way I'm supposed to execute `wget`?

**Comment by DrBanana419** [comment]
Ok I got it, if you copy the above code (fixing the syntax issue), name it flake.nix and run `nix build .#wget`, you get a binary called result/bin/wget, which you can run. 
So now my question is, how do I use it in my configuration.nix?
